docs/_instructions/connection-settings.md

```diff
@@ -35,30 +39,31 @@ This file will contain the detailed trace output
 
 The settings files have the following structure:
 
-	<settings>
-	  <proxies>
-	    <proxy>
-	    <id/>
-	      <active>        'true' | 'false' </active>
-	      <protocol>      'http'| 'direct' | 'socks' </protocol>
-	      <host>          domain name </host>
-	      <port>          port to use </port>
-	      <username>      user name to use for the proxy </username>
-	      <password>      password to use for the proxy </password>
-	      <verify> true | false </verify>
-	      <nonProxyHosts> glob ( '|' glob )* </nonProxyHosts> 
-	    </proxy>
-	  </proxies>
-	  <servers>
-	    <server>
-	      <id>default</id>
-		  <username>username</username>
-	      <password>password</password>
-		  <verify> true | false </verify>
-		  <trust> comma separated paths to X509 certificates </trust>
-	    </server>
-	  </servers>
-	</settings> 
+
+<settings>
```

Comment:
You removed the leading 4 spaces which makes this no longer a block. Please restore.